examples/plot_branin.py

```diff
@@ -1,54 +1,59 @@
+from time import time
+
 import numpy as np
 import matplotlib.pyplot as plt
 
-from skopt import acquisition
-from skopt import gp_minimize
 from skopt.benchmarks import branin
+from skopt.dummy_opt import dummy_minimize
+from skopt.gp_opt import gp_minimize
 
-fig = plt.gcf()
-plt.set_cmap("viridis")
 bounds = [[-5, 10], [0, 15]]
-subplot_no = 311
-
-x1_values = np.linspace(-5, 10, 100)
-x2_values = np.linspace(0, 15, 100)
-x_ax, y_ax = np.meshgrid(x1_values, x2_values)
-vals = np.asarray(list(zip(x_ax.ravel(), y_ax.ravel())))
-
-for i, acq in enumerate(['UCB', 'EI']):
-    res = gp_minimize(
-        branin, bounds, search='sampling', maxiter=200, random_state=0,
-        acq=acq)
-    gp_model = res.models[-1]
-    opt_points = res['x_iters']
-
-    x_opt = None
-    if acq == "EI":
-        x_opt = res.fun
-
-    acquis_values = acquisition(vals, gp_model, bounds=bounds, x_opt=x_opt,
-                                method=acq)
-    acquis_values = acquis_values.reshape(100, 100)
-
-    plt.subplot(subplot_no)
-    plt.pcolormesh(x_ax, y_ax, acquis_values)
-    plt.plot(opt_points[:, 0], opt_points[:, 1], 'ro', markersize=2)
-    plt.colorbar()
-    plt.xlabel('X1')
-    plt.xlim([-5, 10])
-    plt.ylabel('X2')
-    plt.ylim([0, 15])
-    plt.title(acq)
-    subplot_no += 1
-
-plt.subplot(313)
-func_vals = np.reshape([branin(val) for val in vals], (100, 100))
-plt.pcolormesh(x_ax, y_ax, func_vals)
-plt.colorbar()
-plt.xlabel('X1')
-plt.xlim([-5, 10])
-plt.ylabel('X2')
-plt.ylim([0, 15])
-plt.title("Branin function values.")
-
+best_dummy_scores = np.zeros((5, 200))
+best_gp_scores = np.zeros((5, 200))
+n_iterations = range(1, 201)
+
+for random_state in range(5):
+    print("Doing a random search for the minimum.")
+    t = time()
+    dummy_model = dummy_minimize(
+        branin, bounds, maxiter=200, random_state=random_state)
+    print(time() - t)
+    print("Best score obtained, %0.4f" % dummy_model.fun)
+
+    print("Doing a gp-based search for the minimum")
+    t = time()
+    gp_model = gp_minimize(
+        branin, bounds, maxiter=200, random_state=random_state, n_start=1)
+    print(time() - t)
+    print("Best score obtained, %0.4f" % gp_model.fun)
+
+    for j in range(1, 201):
+        best_dummy_scores[random_state, j-1] = np.min(dummy_model.func_vals[:j])
+        best_gp_scores[random_state, j-1] = np.min(gp_model.func_vals[:j])
+
+mean_dummy_scores = np.mean(best_dummy_scores, axis=0)
+mean_gp_scores = np.mean(best_gp_scores, axis=0)
+std_dummy_scores = np.std(best_dummy_scores, axis=0)
```

Comment:
Should we be computing the standard error on the mean? Can someone help with figuring this out?

We show the mean, and are interested in how much the mean can vary, which makes me think `sem`.